docs/guides/deep.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 3
+---
+
+# Deep Folders
+
+`nestjs-i18n` supports translations in deep (nested) folders.
+
+You can use deep (nested) translation folders by setting the loader option `includeDeepFolders: true`.
+
+```ts
+I18nModule.forRoot({
+  loaderOptions: {
+    path: path.join(__dirname, '/i18n/'),
+    includeDeepFolders: true,
```

Comment:
I think the name `recursive` would be better.

Reply:
Hm, not sure if `recursive` is often used in the context of folders. For me, I would wonder if the folder is supposed to contain itself.

![image](https://user-images.githubusercontent.com/48728802/205577030-52a275f0-8942-4d29-affc-a66718c4cfcf.png)
![image](https://user-images.githubusercontent.com/48728802/205577070-e7d8778f-261b-4924-805b-647974800593.png)

Let me know your final decision. 🙂

Reply:
I was thinking of `rm -r` meaning `recursive delete`. Deep sounds kinda vague in my opinion.

Reply:
`includeSubfolders` sounds good as well!

Reply:
Done via https://github.com/toonvanstrijp/nestjs-i18n/pull/419/commits/507217809dcf6e792dc48f7b9446cfe380c06907.